tools/webpack/packages.js

```diff
@@ -0,0 +1,202 @@
+/**
+ * External dependencies
+ */
+const LiveReloadPlugin = require( 'webpack-livereload-plugin' );
+const CopyWebpackPlugin = require( 'copy-webpack-plugin' );
+const postcss = require( 'postcss' );
+
+const { join, basename } = require( 'path' );
+const { get } = require( 'lodash' );
+
+/**
+ * WordPress dependencies
+ */
+const CustomTemplatedPathPlugin = require( '@wordpress/custom-templated-path-webpack-plugin' );
+const LibraryExportDefaultPlugin = require( '@wordpress/library-export-default-webpack-plugin' );
+
+const baseDir = join( __dirname, '../../' );
+
+/**
+ * Given a string, returns a new string with dash separators converedd to
+ * camel-case equivalent. This is not as aggressive as `_.camelCase` in
+ * converting to uppercase, where Lodash will convert letters following
+ * numbers.
+ *
+ * @param {string} string Input dash-delimited string.
+ *
+ * @return {string} Camel-cased string.
+ */
+function camelCaseDash( string ) {
+	return string.replace(
+		/-([a-z])/g,
+		( match, letter ) => letter.toUpperCase()
+	);
+}
+
+module.exports = function( env = { environment: 'production', watch: false } ) {
+	const mode = env.environment;
+	const suffix = mode === 'production' ? '.min': '';
+
+	const gutenbergPackages = [
+		'api-fetch',
+		'a11y',
+		'autop',
+		'blob',
+		'blocks',
+//		'block-library', // Not on npm yet.
+		'block-serialization-default-parser',
+		'block-serialization-spec-parser',
+		'components',
+		'compose',
+		'core-data',
+		'data',
+		'date',
+		'deprecated',
+		'dom',
+		'dom-ready',
+//		'edit-post', // Not on npm yet.
+		'editor',
+		'element',
+//		'escape-html', // Not on npm yet.
+		'hooks',
+		'html-entities',
+		'i18n',
+		'is-shallow-equal',
+		'keycodes',
+//		'list-reusable-blocks', // Not on npm yet.
+		'nux',
+		'plugins',
+		'redux-routine',
+//		'rich-text', // Not on npm yet.
+		'shortcode',
+		'token-list',
+		'url',
+		'viewport',
+		'wordcount',
+	];
+
+	const packagesStyles = [
```

Comment:
I noticed that in Gutenberg we don't have this second variable and we just rely on this line https://github.com/Yoast/wordpress-develop-mirror/pull/118/files#diff-df0280933db62a33cd94973b5ec4871eR167 to know whether or not there are files to move and minify.

Any particular reason for this here? I'm thinking this may result in some CSS to be missed if we ever add CSS to another package.

Reply:
I tested this before and it didn't work, but now that I test it it works. I agree that that's more elegant, considering the case a package adds styles.